content/en/opentelemetry/collector_exporter/deployment.md

```diff
@@ -251,6 +288,53 @@ To deploy the OpenTelemetry Collector and Datadog Exporter in a Kubernetes Gatew
          key: ${env:DD_API_KEY}
    # ...
    ```
+1. Configure host metadata collection:
+   In a gateway deployment, you need to ensure that host metadata is collected by the agent collectors and preserved by the gateway collector.
+
+   **Agent collector configuration**:
+
+   ```yaml
+   processors:
+     resourcedetection:
+       detectors: [system, env]
+     k8sattributes:
+       passthrough: true
+
+   exporters:
+     otlp:
+       endpoint: "<GATEWAY_HOSTNAME>:4317"
+
+   service:
+     pipelines:
+       traces:
+         receivers: [otlp]
+         processors: [resourcedetection, k8sattributes, transform, batch]
+         exporters: [otlp]
+   ```
+
+   **Gateway collector configuration**:
+
+   ```yaml
+   processors:
+     k8sattributes:
+       extract:
+         metadata: [node.name, k8s.node.name]
+
+   exporters:
+     datadog:
+       api:
+         key: ${DD_API_KEY}
+       hostname_source: resource_attribute
+
+   service:
+     pipelines:
+       traces:
+         receivers: [otlp]
+         processors: [k8sattributes, batch]
+         exporters: [datadog]
+   ```
+
+   This ensures that host metadata is collected by the agents and properly forwarded through the gateway to Datadog. For more information, see [Mapping OpenTelemetry Semantic Conventions to Infrastructure List Host Information][14].
```

Comment:
It might make sense to move at least the first sentence here to above the example right after the "what" because it explains the "why", but up to you.

Reply:
Thanks! Good suggestion.